pkg/kubernetes/standalone/src/cockpit-kube-launch/main.go

```diff
@@ -70,8 +70,12 @@ func setupCertificates() {
 		log.Fatalf("Couldn't create certificate directory %s", dErr)
 	}
 
-	// Setting ownership on the certificate may fail
-	// so execute ensure and the check results without
+	// Finding these certificate files or setting ownership on
+	// the certificate may fail so execute combine and ensure
+	// and the check results without
+	exec.Command("/usr/sbin/remotectl", "certificate",
+		"/var/run/secrets/ws-certs.d/tls.crt",
+		"/var/run/secrets/ws-certs.d/tls.key").CombinedOutput()
```

Comment:
This will fail if valid cert files aren't present. It doesn't fall back to generating a self-signed one if it fails. Which i think makes sense. That's why ensure is a separate call below.